docroot/themes/humsci/humsci_basic/src/js/shared/carousel-slides/carousel-slides-height.js

```diff
@@ -0,0 +1,52 @@
+const slides = document.querySelectorAll('.paragraph--type--hs-carousel');
+let timeOutFunctionId; // a numeric ID which is used by clearTimeOut to reset the timer
+
+if (slides.length > 0) {
+  restrictHeight();
+
+  // Watch for when the browser window resizes, then run the restrictHeight
+  // function to reset the height of the text boxes
+  window.addEventListener('resize', function() {
+    clearTimeout(timeOutFunctionId);
+    timeOutFunctionId = setTimeout(restrictHeight, 250);
+  });
+}
+
+// Set the height of all text boxes within a Carousel to that
+// of the tallest text box
+function restrictHeight() {
+  for (let i = 0; i < slides.length; i++) {
```

Comment:
The [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) loop is a bit easier to read and we won't need to declare all of these counter variables.

Reply:
For some reason I thought forEach was not supported in IE and chose for loops instead. forEach are definitely easier to implement. :smile: